CONTRIBUTING.rst

```diff
@@ -76,7 +76,21 @@ Before you submit a pull request, check that it meets these guidelines:
 
 1. The pull request should include tests.
 2. If the pull request adds functionality, the docs should be updated. Put your new functionality into a function with a docstring.
-3. If checks to not pass, have a look at https://travis-ci.com/pyfar/pyfar/pull_requests for more information.
+3. If checks do not pass, have a look at https://app.circleci.com/pipelines/github/pyfar/pyfar for more information.
+
+Function and Class Guidelines
+-----------------------------
+
+pyfar functions and classes should raise errors if
+
+- audio objects do not have the correct type (e.g. a TimeData instance is passed but a Signal instance is required)
```

Comment:
Do we or should we have guidelines on checking the type of parameters?
E.g. in case of bools, if private properties of classes are set

Reply:
I think I read somewhere that the general Python recommendation is to document the expected input type but not explicitly check it. This, we would already follow.